tests/conversion

```diff
@@ -255,7 +262,7 @@ if hasNeededAPIExtension instance_import_conversion; then
     wget -q -O "${IMAGE_PATH}" https://cloud-images.ubuntu.com/releases/24.04/release/ubuntu-24.04-server-cloudimg-amd64.vmdk
 
     conversion_vm vm-u24-vmdk-dir dir "${IMAGE_PATH}" "yes"
-    conversion_vm vm-u24-vmdk-zfs zfs "${IMAGE_PATH}" "yes"
```

Comment:
A few lines above we test `zfs` and `btrfs` with Alpine QCoW2 and CentOS 9-Stream respectively. Therefore, I've used `dir` and `lvm` for VMDK image (instead of using zfs again). 

VMDK image is also tested with `dir` and `zfs` using *zfs backups volume* a few lines below.